src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netstandard1.6/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Does this file end up in a NuGet package that ships to customers? If so, don't put any special header in here.